python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -0,0 +1,414 @@
+        doc="Set of preprocessed postage stamps, each centered on a single bright star.",
+        name="brightStarStamps",
+        storageClass="BrightStarStamps",
+        dimensions=("visit", "detector")
+    )
+
+
+class ProcessBrightStarsConfig(pipeBase.PipelineTaskConfig,
+                               pipelineConnections=ProcessBrightStarsConnections):
+    """Configuration parameters for ProcessBrightStarsTask
+    """
+    magLimit = pexConfig.Field(
+        dtype=float,
+        doc="Magnitude limit, in Gaia G; all stars brighter than this value will be processed",
+        default=18
+    )
+    stampSize = pexConfig.ListField(
+        dtype=int,
+        doc="Size of the stamps to be extracted, in pixels",
+        default=(250, 250)
+    )
+    modelStampBuffer = pexConfig.Field(
+        dtype=float,
+        doc="'Buffer' factor to be applied to determine the size of the stamp the processed stars will "
+            "be saved in. This will also be the size of the extended PSF model.",
+        default=1.1
+    )
+    warpingKernelName = pexConfig.ChoiceField(
+        dtype=str,
+        doc="Warping kernel",
+        default="lanczos5",
+        allowed={
+            "bilinear": "bilinear interpolation",
+            "lanczos3": "Lanczos kernel of order 3",
+            "lanczos4": "Lanczos kernel of order 4",
+            "lanczos5": "Lanczos kernel of order 5",
+        }
+    )
+    annularFluxRadii = pexConfig.ListField(
+        dtype=int,
+        doc="Inner and outer radii of the annulus used to compute the AnnularFlux for normalization, "
+            "in pixels.",
+        default=(40, 50)
+    )
+    annularFluxStatistic = pexConfig.ChoiceField(
+        dtype=str,
+        doc="Type of statistic to use to compute annular flux.",
+        default="MEANCLIP",
+        allowed={
+            "MEAN": "mean",
+            "MEDIAN": "median",
+            "MEANCLIP": "clipped mean",
+        }
+    )
+    numSigmaClip = pexConfig.Field(
+        dtype=float,
+        doc="Sigma for outlier rejection; ignored if annularFluxStatistic != 'MEANCLIP'.",
+        default=4
+    )
+    numIter = pexConfig.Field(
+        dtype=int,
+        doc="Number of iterations of outlier rejection; ignored if annularFluxStatistic != 'MEANCLIP'.",
+        default=3
+    )
+    badMaskPlanes = pexConfig.ListField(
+        dtype=str,
+        doc="Mask planes that, if set, lead to associated pixels not being included in the computation of the"
+            " annular flux.",
+        default=('BAD', 'CR', 'CROSSTALK', 'EDGE', 'NO_DATA', 'SAT', 'SUSPECT', 'UNMASKEDNAN')
+    )
+    refCatLoader = pexConfig.ConfigurableField(
+        target=LoadIndexedReferenceObjectsTask,
+        doc="reference object loader for astrometric calibration",
+    )
+
+    def setDefaults(self):
+        self.refCatLoader.ref_dataset_name = "gaia_dr2_20200414"
+
+
+class ProcessBrightStarsTask(pipeBase.PipelineTask, pipeBase.CmdLineTask):
```

Comment:
I'd rather not just yet, if only because:

a) I am (and will keep on) running this on RC2 a lot;
b) the Gaia refcat is not yet included in any gen3 repos (as far as I know).

Happy to clean things up when the time comes though!